src/components/App.js

```diff
@@ -43,14 +47,12 @@ class App extends React.Component {
         <p>
           Read about the process on the <strong>Dash Newsroom</strong>
           : <br />
-          <a href={announcmentUrl}>{announcmentUrl}</a>
+          <a href={announcementUrl}>{announcementUrl}</a>
```

Comment:
Good catch!

Reply:
i think the envvar has the same typo, but I didn't want to update that and risk breaking the deployments